### PR TITLE
feat: [SRM-11016]: Add default Timeline window for all periods

### PR DESCRIPTION
**Pull request by vasuki-kalluri**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by vasuki-kalluri** [comment]
image

**Comment by vasuki-kalluri** [comment]
retrigger jest

**Comment by vasuki-kalluri** [comment]
retrigger eslint

**Comment by vasuki-kalluri** [comment]
retrigger jest

**Comment by vasuki-kalluri** [comment]
retrigger titlecheck

File: src/modules/85-cv/components/ChangeTimeline/components/TimelineSlider/TimelineSlider.tsx [hunk 1]
```diff
@@ -115,6 +116,18 @@ const TimelineSlider: React.FC<TimelineSliderProps> = ({
     }
   }, [hideSlider, initialSliderWidth])
 
+  useEffect(() => {
+    if (selectedTimePeriod) {
+      const offset = containerWidth - initialSliderWidth
```
Comment (harshilbits) [inline src/modules/85-cv/components/ChangeTimeline/components/TimelineSlider/TimelineSlider.tsx:121]: Why do we need this ?

Comment (vasuki-kalluri) [reply src/modules/85-cv/components/ChangeTimeline/components/TimelineSlider/TimelineSlider.tsx:121]: This is to add a default Slider for the time periods.

**Review by deepesh-ui — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Pull request merged**